docs/materials/day4/part2-ex2-file-transfer.md

```diff
@@ -54,7 +54,7 @@ In our first blast job from yesterday, the database files in the `pdbaa` directo
     A typical command for creating a tar file is:
 
         :::console
-        user@training $ tar cvzf [compressed filename] [list of files]
+        user@training $ tar cvzf [compressed filename] <LIST OF FILES OR DIRECTORIES>
```

Comment:
Can we make this `<COMPRESSED FILENAME>`, too?

Also, all the angle bracket things should be `%RED%<THING>%ENDCOLOR%` to be consistent